js/scenes/Map.js

```diff
@@ -43,8 +59,17 @@ const styles = StyleSheet.create({
     fontSize: 22,
     textAlign: 'center',
   },
-
   text: {
     ...defaultTextStyle,
   },
+  container: {
+    ...StyleSheet.absoluteFillObject,
+    height: 300,
+    width: 400,
+    justifyContent: 'flex-end',
+    alignItems: 'center',
+  },
+  map: {
+    ...StyleSheet.absoluteFillObject,
+  },
 });
```

Comment:
+1 @wilsonpage commented about the options of styling the map (not styles in react, but the options that google give to style the colors ,etc.).

Ideally we should have this in a separate component, even a separate repo.